### ISSUE TITLE
Error Handling Image stored in longblob  field

### ISSUE THREAD

**Issue by barclay2506** [zim32/mysql.dart#61]
I am trying to fetch an image stored in longblob field  from mysql and throws an error that puts:
 "FormatException: Invalid UTF-8 byte (at offset 0)"
the thing is that i used before another package (galileo_sqljocky5) and i could fetch the data without any problem, so i dont know why 
the error here, i have fetched the same image from c# and visual basic with their respective MysqlConnector so the 
image isnt the problem i guess.
there is an image with the field definition,another with the error, another one with the code that causes it, if you need any other information let me know
![field definition](https://github.com/zim32/mysql.dart/assets/125677157/95f12324-d6cf-461b-ad62-8b81e1b70734)

![code_error_mysql_utils](https://github.com/zim32/mysql.dart/assets/125677157/7083fac1-dd54-491f-9527-bc045aa3167c)

![error mysql_utils](https://github.com/zim32/mysql.dart/assets/125677157/5a70d3ce-29f3-4d82-ad1e-0fe4a52359df)

**Comment by unleed-l** [comment]
any update about that?

**Comment by barclay2506** [comment]
nah, still the same, had to use antoher package just to handle blob data in general, with galileo_sqljocky5 you can do it

**Comment by unleed-l** [comment]
> nah, still the same, had to use antoher package just to handle blob data in general, with galileo_sqljocky5 you can do it

Bro, i'm trying to use galileo_sqljocky5, but connection throws this error:
`flutter: SocketException: Socket has been closed`

Do you know how can i fix?

**Comment by insinfo** [comment]
I forked this driver and updated it and fixed several bugs https://github.com/insinfo/mysql.dart
